com.vladfaust.unitywakatime/com.vladfaust.unitywakatime.asmdef

```diff
@@ -0,0 +1,3 @@
+﻿{
+	"name": "NewAssembly"
```

Comment:
Any comments on why using `NewAssembly`? Maybe leaving blank? I'm not strong in C#...

Reply:
Good question. This is just the internal name that Unity has given to the thing. Not sure what the contents of this file are. I'll investigate a bit more.

Reply:
Try to import into a project without this file

Reply:
Technically we probably can make do without the file, but it's [recommended by most people](https://gist.github.com/LotteMakesStuff/6e02e0ea303030517a071a1c81eb016e#file-5-md) and a good idea to have them there as using assembly definition files should [increase build performance](https://blogs.unity3d.com/2017/11/22/unity-2017-3b-feature-preview-assembly-definition-files-and-transform-tool/).

I've given it a unique name.